From 67a1c7224948ca5a31c57abb2a0a958a8bc4f45c Mon Sep 17 00:00:00 2001
From: Dinesh Kumar Mirche <dmirche@habana.ai>
Date: Thu, 25 Jul 2024 05:31:52 +0300
Subject: [PATCH] [SW-195702] gaudi3: export apis to open jtag

Export security_handle_jtag_if and security_rr_open_debug_if_config
ROM api so that FW components can use this apis to open the JTAG IF.

Change-Id: I44479af8a040c98c6fb1b2cf4a31b3ef7595e553
Signed-off-by: Dinesh Kumar Mirche <dmirche@habana.ai>
---

diff --git a/arch/gaudi3/arc/arcboot_rom.txt b/arch/gaudi3/arc/arcboot_rom.txt
index 102840c..17730df 100644
--- a/arch/gaudi3/arc/arcboot_rom.txt
+++ b/arch/gaudi3/arc/arcboot_rom.txt
@@ -34,3 +34,5 @@
 pcie_phy_set_sram_ext_ld_done
 pcie_set_pcie_device_vendor_id
 sha384_csum
+security_handle_jtag_if
+security_rr_open_debug_if_config
